src/web_app/js/peerconnectionclient.js

```diff
@@ -174,9 +174,9 @@ PeerConnectionClient.prototype.getPeerConnectionStats = function(callback) {
 
 PeerConnectionClient.prototype.doAnswer_ = function() {
   trace('Sending answer to peer.');
-  this.pc_.createAnswer(this.setLocalSdpAndNotify_.bind(this),
-      this.onError_.bind(this, 'createAnswer'),
-      PeerConnectionClient.DEFAULT_SDP_CONSTRAINTS_);
+  this.pc_.createAnswer()
```

Comment:
hrm... that should hopefully work with https://github.com/webrtc/adapter/blob/master/adapter.js#L228-229
now if firefox doesn't error i can fix that.